src/sentry/releasehealth/metrics.py

```diff
@@ -340,3 +346,72 @@ def _count_users(total: bool, referrer: str) -> Dict[Any, int]:
             rv[project_id, release] = adoption
 
         return rv
+
+    def check_has_health_data(
+        self, projects_list: Sequence[Union[ProjectId, ProjectRelease]]
+    ) -> Set[Union[ProjectId, ProjectRelease]]:
+        now = datetime.now(pytz.utc)
+        start = now - timedelta(days=3)
+
+        projects_list = list(projects_list)
+
+        if len(projects_list) == 0:
+            return set()
+
+        includes_releases = type(projects_list[0]) == tuple
```

Comment:
```suggestion
        includes_releases = isinstance(projects_list[0], tuple)
```

I believe this may make some type: ignore obsolete